eprihlaska/__init__.py

```diff
@@ -34,7 +34,14 @@
 csrf.init_app(app)
 babel = Babel(app)
 app.config['BABEL_DEFAULT_LOCALE'] = 'sk_SK'
-locale.setlocale(locale.LC_ALL, 'sk_SK.utf8')
+try:
+   locale.setlocale(locale.LC_ALL, 'sk_SK.utf8')
+except Exception:
```

Comment:
You should specify the exact exception which can happen. Also avoid passing exception handling.
For better explanation see this https://stackoverflow.com/a/21553825/1442465